infra/scripts/focused-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Runs go test on any packages that differ from vmware/master that have
+# Go test files.
+#
+# Usage: infra/scripts/focused-test.sh
+#
+
+echo "Finding modified packages with test files"
+PKG=$(git diff --stat vmware/master --name-only | xargs dirname | uniq | while read i; do
```

Comment:
Should figure out a generic remote. There are definitely cases where diffing against master will be incorrect - if for example we're working with a feature branch.
I'll likely make the remote an environment variable with a default. I had considered just upstream branch, but that's not always set and not necessarily what I wanted for this.

I wanted this to test those changes that would be present in a PR vs `vmware/vic` which is where the tests tend to fail.